README.md

```diff
@@ -202,6 +202,8 @@ vsi_interface | string | Set it as "public" to create a Floating IP to connect t
 vsi_user_data_file | string | User data to be made available when setting up the virtual server instance. Optional.
 | OR |
 vsi_user_data | string | User data to be made available when setting up the virtual server instance. Optional. This is the string input variable.
+vsi_boot_vol_capacity | string | The capacity to use for the volume (in gigabytes). Must be at least the image's minimum_provisioned_size. The maximum value may increase in the future. Optional.
+vsi_boot_vol_profile | string | User can provide the available profile for volume attachments. Refer https://cloud.ibm.com/docs/vpc?topic=vpc-block-storage-profiles&interface=ui for profile info. Optional.
```

Comment:
list of allowed values needs to be mentioned here. Document does not list the allowed values